goreleaser.yaml

```diff
@@ -45,16 +100,17 @@ changelog:
     exclude:
     - '^tests:'
 
-brews:
-  - tap:
-      owner: deepsourcelabs
-      name: homebrew-cli
-      branch: cli-release
-      token: "{{ .Env.HOMEBREW_TOKEN }}"
-    commit_author:
-      name: deepsourcebot
-      email: bot@deepsource.io
-    homepage: "https://github.com/deepsourcelabs/cli"
-    description: "Command line interface to DeepSource"
-    license: "BSD 2-Clause Simplified License"
-    skip_upload: auto
+# TODO: fix homebrew tap (siddhant-deepsource)
+# brews:
+#   - tap:
+#       owner: deepsourcelabs
+#       name: homebrew-cli
+#       branch: cli-release
+#       token: "{{ .Env.HOMEBREW_TOKEN }}"
+#     commit_author:
+#       name: deepsourcebot
+#       email: bot@deepsource.io
+#     homepage: "https://github.com/deepsourcelabs/cli"
+#     description: "Command line interface to DeepSource"
+#     license: "BSD 2-Clause Simplified License"
+#     skip_upload: auto
```

Comment:
@siddhant-deepsource Logs: https://github.com/burntcarrot/cli/runs/7248243117?check_suite_focus=true#step:4:97

Reply:
Yep, I'll take a look today. Looks like an easy fix. The real trouble is the variable expansion for resolving the environment variables. Its failing on linux environment.

Reply:
Have added id to homebrew tap workflow in [10dc8bd](https://github.com/deepsourcelabs/cli/pull/185/commits/10dc8bd3e5e3ad7338426b0697beb19d629830f5). Can you retest once?

Reply:
Works fine. Closing for now.